SDALib-CPP/SDALib-CPP/SDA_portability.h

```diff
@@ -7,6 +7,8 @@
 #pragma once
 
 #include "ipclib_portability.h"
+#define _SILENCE_EXPERIMENTAL_FILESYSTEM_DEPRECATION_WARNING 1
+#include <experimental/filesystem>
```

Comment:
add `namespace filesystem = std::experimental::filesystem;`

Also these same defines are currently in `SDATypesConverter.h`, from which they can be removed